about.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script>
+      src="https://kit.fontawesome.com/cb6c5078b1.js"
+      crossorigin="anonymous"
+      </script>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+  <link rel="stylesheet" href="about.css">
+  <title>About Burning Circle Flight Academy</title>
+</head>
+<body>
+  <div class="top-section">
+    <div class="bar-container">
+    <ul class="bars">
+      <li>
+        <i class="fab fa-facebook-f"></i>
+      </li>
+      <li>
+        <i class="fab fa-twitter"></i>
+      </li>
+      <li>
+        <h4>English</h4>
+      </li>
+      <li>
+        <h4>My Page</h4>
+      </li>
+      <li>
+        <h4>Logout</h4>
+      </li>
+    </ul>
+    </div>
+  </div>
+  <header>
+    <div class="navbar">
+        <div class="logo-DT">
+          <a href="index.html">BCFA</a>
+          <div class="menu-bar nav-toggle" id="menu">
+            <i class="fas fa-bars fa-3x"></i>
+          </div>
+          <div class="cancel" id ='close'>
+            <p>&times;</p>
+          </div>
+        </div>
+
+        <nav class="menu-list" id="nav-mobile">
+          <ul class="items">
+            <li class="home">
+              <a href="index.html">Home</a>
+            </li>
+
+            <li class="links">
+              <a href="about.html">About</a>
```

Comment:
* Kindly fix the menu bar for the about page mobile view it is currently not visible 
![image](https://user-images.githubusercontent.com/52497246/133898170-7d7fbb71-693d-4d2d-9c94-039a1ba49866.png)